lib/DiscoveryStreamFeed.jsm

```diff
@@ -851,9 +851,10 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
    */
   async refreshAll(options = {}) {
     const { updateOpenTabs, isStartup } = options;
+    const dispatchQueue = [];
     const dispatch = updateOpenTabs
       ? action => this.store.dispatch(ac.BroadcastToContent(action))
-      : this.store.dispatch;
+      : e => dispatchQueue.push(e);
```

Comment:
Let's keep the naming consistent, so `e` -> `action` and could probably just move it to a single function, e.g.,
```js
const dispatch = action => updateOpenTabs ? store…(action) : queue…(action);
```

Reply:
:+1:

Reply:
You have a huge opinion if I keep it as two functions?

```
    const dispatch = updateOpenTabs
      ? action => this.store.dispatch(ac.BroadcastToContent(action))
      : action => dispatchQueue.push(action);
``

Reply:
Either is fine. One saves an `action =>` 9 characters 😛 whatever you think is easier to read